m4opt/models/bandpass/bandpass.py

```diff
@@ -0,0 +1,115 @@
+from astropy.modeling.models import Tabular1D
+from astropy import units as u
+from astropy.units.physical import get_physical_type
+
+from .core import BaseBandpass
+
+# TODO: remove check below when tynt is released
+# and add tynt to required dependencies
+try:
+    from tynt import FilterGenerator
+    tynt_filters = FilterGenerator().available_filters()
+except ImportError:
+    tynt_filters = None
```

Comment:
It's not clear `tynt` will *ever* get a true release, as it's slated to be added to synphot - see https://github.com/spacetelescope/synphot_refactor/pull/257 .  So as I see it for now there's two options:
1.  add the *specific git commit of tynt* to the `setup.cfg` file and treat that as a hard dependency for now. You can see how to do that here: https://python-packaging.readthedocs.io/en/latest/dependencies.html#packages-not-on-pypi
2. Have a clearer optional dependency message - see my immediately following suggestion: